<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>751</width>
     <height>611</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 240, 254)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="login">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>519</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover{
	background-color:white;
	border-top-left-radius: 10px;
	border-bottom-left-radius: 10px;
border-top-right-radius: 10px;
	border-bottom-right-radius: 10px;

}

QPushButton{
background-color: rgb(255, 207, 233);
border-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(0, 0, 0, 0), stop:0.52 rgba(0, 0, 0, 0), stop:0.565 rgba(82, 121, 76, 33), stop:0.65 rgba(159, 235, 148, 64), stop:0.721925 rgba(255, 238, 150, 129), stop:0.77 rgba(255, 128, 128, 204), stop:0.89 rgba(191, 128, 255, 64), stop:1 rgba(0, 0, 0, 0));
border-radius:30px;
	border-top-left-radius: 10px;
	border-bottom-left-radius: 10px;
border-top-right-radius: 10px;
	border-bottom-right-radius: 10px;

}</string>
   </property>
   <property name="text">
    <string> Login</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="userNameField">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>300</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0, 0);
font: 11pt &quot;Nirmala UI&quot;;
gridline-color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="cursorMoveStyle">
    <enum>Qt::LogicalMoveStyle</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordField">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>370</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0, 0);
font: 11pt &quot;Nirmala UI&quot;;
gridline-color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>310</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Narrow</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>username :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>380</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Narrow</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>password :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>100</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>661</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>true</underline>
     <strikeout>false</strikeout>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Nirmala UI&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#aa557f;&quot;&gt;please enter a correct username and password &lt;/span&gt;&lt;span style=&quot; font-weight:600; color:#aa557f;&quot;&gt;to login &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="error">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>420</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>460</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Narrow</family>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; color:#ff55ff;&quot;&gt;dont have account?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createAcc">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>450</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Narrow</family>
     <pointsize>10</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:30px;
	background-color: rgb(255, 246, 251);
}
QPushButton:hover{
	background-color:white;
	border-top-left-radius: 10px;
	border-bottom-left-radius: 10px;
border-top-right-radius: 10px;
	border-bottom-right-radius: 10px;

}

QPushButton{
border-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(0, 0, 0, 0), stop:0.52 rgba(0, 0, 0, 0), stop:0.565 rgba(82, 121, 76, 33), stop:0.65 rgba(159, 235, 148, 64), stop:0.721925 rgba(255, 238, 150, 129), stop:0.77 rgba(255, 128, 128, 204), stop:0.89 rgba(191, 128, 255, 64), stop:1 rgba(0, 0, 0, 0));
border-radius:30px;
	border-top-left-radius: 10px;
	border-bottom-left-radius: 10px;
border-top-right-radius: 10px;
	border-bottom-right-radius: 10px;

}</string>
   </property>
   <property name="text">
    <string>create new account</string>
   </property>
  </widget>
  <widget class="QLabel" name="loginICON">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>90</y>
     <width>441</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="login.qrc">:/login_icon/icons/loginn.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>731</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>751</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 221, 246);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="login.qrc"/>
  <include location="icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
